test/functorch/test_aotdispatch.py

```diff
@@ -285,24 +286,20 @@ def is_in_base(t, maybe_tensors):
     return False
 
 
-def skipIfDynamoInput(reason, xfail=False):
+def skipIfDynamoInput(reason):
```

Comment:
My previous PR didn't actually test xfail properly here, it was just skipping them anyway. It turns out in python 3 you can't xfail tests from methods, only at the class level. So I implemented expected failure with xfail_inherited_tests instead.

Reply:
Actually let me just move this into its own PR